package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "adage-frontend",
-  "version": "0.1.22",
+  "version": "0.1.23",
   "dependencies": {
+    "@stdlib/stdlib": "^0.0.90",
```

Comment:
Has there been a discussion about including `stdlib`? It looks really cool and potentially a good asset for this project. It also looks very large and complex. There are 129 dependencies, for example. Do we know how much this inclusion will expand our download size? I'm not up to speed on changes to our build environment, so maybe there's a modular build process that keeps things to just what we're using. Let me know if these concerns are unwarranted or have already been discussed and accepted elsewhere.

Reply:
The whole `stdlib` is massive, yes, but the functions are imported individually, so nothing extraneous should be included. I would also suspect that d3 would dwarf the size of anything else we're including anyway.

Just ran a build for this PR. `main.js` is 778.5kb, which is about what it has been before.

Reply:
Nice. The "kitchen sink" aspect of `stdlib` was making me a bit nervous, so "the functions are imported individually" is what I was hoping you would say. 👍

Reply:
Also: webworkers = 👍